prometheus_utils.lua

```diff
@@ -0,0 +1,97 @@
+--
```

Comment:
If you are adding a new source file, I believe the following configuration files will need to be adjusted:
- nginx-lua-prometheus-*.rockspec
- .github/workflows/tests.yml

But honestly I would suggest to try to keep everything in `prometheus.lua` and not bring additional files with different licenses.

Reply:
update